src/cr/cube/subtotal.py

```diff
@@ -43,10 +46,15 @@ def anchor(self):
     def _all_dim_ids(self):
         return [el.get('id') for el in self._dim.elements(include_missing=True)]
 
+    def _validate_hs_ids(self, hs_ids):
+        element_ids = [el['id'] for el in self._dim.elements()]
+        return any(arg in element_ids for arg in hs_ids)
+
     @lazyproperty
     def args(self):
         '''Get H&S args.'''
-        if self.is_valid:
+        hs_ids = self._data.get('args', None)
+        if hs_ids and self.is_valid and self._validate_hs_ids(hs_ids):
```

Comment:
`and self._validate_hs_ids(hs_ids)` is redundant, as that check is done in `is_valid`.